ha/k8s_setup/ha_setup.py

```diff
@@ -269,6 +269,10 @@ def process(self):
             event_manager.subscribe(const.EVENT_COMPONENT, [SubscribeEvent(const.POD_EVENT, ["online", "failed"])])
             Log.info(f'event_manager subscription for {const.EVENT_COMPONENT}\
                        is successful for the event {const.POD_EVENT}')
+            event_manager.subscribe(const.EVENT_COMPONENT, [SubscribeEvent(const.DISK_EVENT, ["online", "failed"])])
```

Comment:
For CVG?

Reply:
Ok..From the comments. I got it that CVG subscription not to be done.. Resolving hence.